docs/contract.rst

```diff
@@ -16,12 +16,15 @@ Contract
 
 .. autoclass:: InvokeResult
     :exclude-members: __init__, __new__
-    :members: wait_for_acceptance, wait_for_acceptance_sync
+    :members: wait_for_acceptance, wait_for_acceptance_sync, contract, invoke_transaction, hash, status, block_number
+    :member-order: groupwise
 
 .. autoclass:: DeployResult
     :exclude-members: __init__, __new__
-    :members: wait_for_acceptance, wait_for_acceptance_sync
+    :members: wait_for_acceptance, wait_for_acceptance_sync, deployed_contract, hash, status, block_number
+    :member-order: groupwise
 
 .. autoclass:: DeclareResult
     :exclude-members: __init__, __new__
-    :members: deploy, deploy_sync, wait_for_acceptance, wait_for_acceptance_sync
+    :members: deploy, deploy_sync, wait_for_acceptance, wait_for_acceptance_sync, class_hash, compiled_contract, hash, status, block_number
+    :member-order: groupwise
```

Comment:
Maybe we should consider setting the member-order to `bysource`, as suggested [here](https://github.com/software-mansion/starknet.py/pull/544).
I think having the attribute members just below the class definition is more natural.

Reply:
I tried bysource and for some reason it was like: properties from class, methods and inherited methods, inherited properties.

So it'd put hash etc.  after wait_for_acceptance but class_hash before wait_for_acceptance (in DeclareResult case)

Reply:
I'll investigate this

Reply:
Okay, let me try that

Reply:
It works except for the DeclareResult.

Code like this

```rst
.. autoclass:: DeclareResult
    :exclude-members: __init__, __new__
    :members: class_hash, compiled_contract, hash, status, block_number, deploy, deploy_sync, wait_for_acceptance, wait_for_acceptance_sync
    :member-order: bysource
```

produces the output like

![image](https://user-images.githubusercontent.com/52135326/208685200-4f70d497-cfc7-44e4-bae1-8c408f751e5c.png)

Reply:
Having it `groupwise` places methods on top, but at least properly separates them from the dataclass fields which I think is better.